keras/preprocessing/image.py

```diff
@@ -479,6 +479,16 @@ def __init__(self,
             raise ValueError('`zoom_range` should be a float or '
                              'a tuple or list of two floats. '
                              'Received arg: ', zoom_range)
+        if zca_whitening:
+            if not featurewise_center:
+                raise ValueError('This ImageDataGenerator specifies '
```

Comment:
Why not turn this into a warning? Seems preferable, if only to avoid breaking existing code that might do this.